### ISSUE TITLE
Update styling of full-text result snippets section

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/exhibits#1066]
A forthcoming ticket will suggest adding a "Preview PDF" toggle to the search results page just below the existing "Preview matches in document text" toggle:

----
<img width="360" alt="sabino_-_virtual_tribunals__test__-_online_exhibits_search_results" src="https://user-images.githubusercontent.com/101482/35464970-3316eb50-02b6-11e8-882d-6644465800ef.png">

----

Thinking about how the two toggles will coexist, we might want to make the following updates to the existing full-text toggle styling and interaction:
 
* Change the label to "Sample matches in document text" (Note 1 in second mockup below; makes more clear that matches shown are not necessarily *all* matches in document). 
* Update the styling of the text to use the default body text styling (might not have to define in this way, but below is what we want the styling to be):
```
color: $gray-dark;
font-size: $font-size-base;
```
* Use a disclosure triangle icon rather than the current chevron icon (Note 2 in second mockup below). The text and icon should both open and close the matches section. The look and interaction should be similar to the Table of contents field's Show link on the Item Show page ([example](https://exhibits.stanford.edu/parker/catalog/ky326yt7099)).
* Left-align the toggle text with the result item title. It's almost aligned now, except there is 15px of left padding. Removing the left padding would make the correct alignment (though this might be more complicated because the padding is added on the document-metadata `dl`; given the PDF preview will need the same alignment, I wonder if moving these toggles outside of the `dl` might make things easier?).
* When the full-text snippet section is open, add a left border to more clearly delineate the snippet section (Note 3 in second mockup below). Here’s some pseudo styling for the border:
```
dd.blacklight-full_text_tesimv { 
    border-left: 1px solid #ddd;
}

dd.blacklight-full_text_tesimv > p {
    margin-left: 12px;
}
```

Here's how the updated page would look when initially loaded:

<img width="446" alt="ft-preview-styling-closed" src="https://user-images.githubusercontent.com/101482/35465334-7a4f3660-02b8-11e8-892e-ee155226f188.png">

And when a preview full-text section is open:

<img width="453" alt="ft-preview-styling-open" src="https://user-images.githubusercontent.com/101482/35465340-87602f3a-02b8-11e8-8763-4a975b579499.png">

**Comment by jkeck** [comment]
One thing that we'll need to consider is that anybody who has saved their metadata configurations since we rolled out the full-text highlight field initially will get the label text that we rolled out with originally.  FWIW, only Feigenbaum actually has full-text indexed into it, so it's safe to say nobody is rending this field currently.  However; if an exhibit that currently exists saves its metadata configuration, and then gets full text indexed later, they'll have the old label.  This is probably not a huge concern, but will be the reason if we have a discrepancy between labels in exhibits.
